src/database/seeders/BouncerSeeder.php

```diff
@@ -45,4 +47,50 @@ private function createAdminAccount()
         }
         return $user;
     }
+
+    private function createRolePermission() {
+
+        Bouncer::allow('student')->to('view-profile');
+        Bouncer::allow('student')->to('edit-profile');
+        Bouncer::allow('student')->to('review-my-quizzes');  
+        Bouncer::allow('student')->to('conduct-quizzes');
```

Comment:
From [`task: endpoint validators` PR](https://github.com/CS48X0-Behavior-Tool-Project/behavior-rating-tool/pull/64/files): 

```php
Bouncer::allow('admin')->to(['delete', 'edit', 'create', 'view'], [User::class, Video::class, Quiz::class]);
```

```php
Bouncer::allow('<role>')->to(['list', 'of', 'abilities'], ['list', 'of', 'models', 'to', 'apply', 'abilities']);
```

Note: may need to `use path/to/Model` in order to enable the list of models to apply the abilities to.

Therefore:
```php
Bouncer::allow('student')->to(['view-profile', 'edit-profile', 'review-my-quizzes', 'conduct-quizzes''], [User::class, Quiz::class, Attempt::class]);
```

Functionally it should be on par, however it's your call. If you think the way it is already is more readable I've no objection.